client/engine/chainservice/eth_chainservice.go

```diff
@@ -53,18 +53,21 @@ func NewEthChainService(chain ethChain, na *NitroAdjudicator.NitroAdjudicator,
 	// Use a buffered channel so we don't have to worry about blocking on writing to the channel.
 	ecs := EthChainService{chain, na, naAddress, caAddress, vpaAddress, txSigner, make(chan Event, 10), logger}
 
-	err := ecs.subcribeToEvents()
-	return &ecs, err
+	go ecs.listenForLogEvents(context.Background())
+	return &ecs, nil
 }
 
 // defaultTxOpts returns transaction options suitable for most transaction submissions
 func (ecs *EthChainService) defaultTxOpts() *bind.TransactOpts {
+	// FEVM only supports type 2 transactions
```

Comment:
This comment, while true, doesn't make sense now that we're copying all the fields.